code/modules/client/client_procs.dm

```diff
@@ -3,6 +3,8 @@
 	////////////
 #define UPLOAD_LIMIT		1048576	//Restricts client uploads to the server to 1MB //Could probably do with being lower.
 
+GLOBAL_LIST_INIT(blacklisted_builds, list(1407 = "bug preventing client display overrides from working leads to clients being able to see things/mobs they shouldn't be able to see"))
```

Comment:
Should be a config option, but whatever

Reply:
Blacklisting builds that are exploits against balance decisions code side are a code decision, not a server one.

Reply:
oh, ok. still